tests/serialize/Makefile

```diff
@@ -36,10 +36,12 @@ t7:
 	diff ,t7.nt t7-ref.nt
 
 	# NQUADS t8 ends up including todays datetime in data so not reproducable
-t8:
-	node ./data.js -in=t5.n3  -format=application/n-quads -dump=,t8.nq
-	diff ,t8.nq t8-ref.nq
-	#   NTriple output
+#t8:
```

Comment:
It is dumping the entire contents of the store including HTTP metadata.  This doesn't work as a test as it contains the date of the document in a header, which always changes.   The test is not included.  There is no test for the HTTP medatadata in hte store, and no spec either AFAIK. so yes an issue would be appropriate.